test/MOI_additional.jl

```diff
@@ -107,6 +107,28 @@ end
     x = MOI.add_variable(optimizer)
     t = MOI.add_constraint(optimizer, x, MOI.ZeroOne())
     @test_throws ErrorException b = MOI.add_constraint(optimizer, x, MOI.Interval(2.0, 3.0))
+
+    MOI.empty!(optimizer)
```

Comment:
Nitpick: These tests make sense, but are not really about binary variables or implicit bounds.
Could you move them down with the other indicator tests?

Reply:
sure yup